Sources/SwiftGen/Commander/ConfigCommands.swift

```diff
@@ -153,7 +131,58 @@ enum ConfigCLI {
   }
 }
 
-// MARK: Private
+// MARK: - Parallel commands
+
+private extension Config {
+  func runCommands(verbose: Bool) throws {
+    let commandsAndEntries = try collectCommandsAndEntries()
+
+    let errors = commandsAndEntries.parallelCompactMap { cmd, entry -> Swift.Error? in
+      do {
+        try run(cmd: cmd, entry: entry, verbose: verbose)
+        return nil
+      } catch {
+        return error
+      }
+    }
+
+    if errors.count == 1 {
+      throw errors[0]
+    } else if errors.count > 1 {
+      throw Error.multipleErrors(errors)
+    }
+  }
+
+  func run(cmd: ParserCLI, entry: ConfigEntry, verbose: Bool) throws {
+    var entry = entry
+
+    entry.makingRelativeTo(inputDir: inputDir, outputDir: outputDir)
+    if verbose {
+      for item in entry.commandLine(forCommand: cmd.name) {
+        logMessage(.info, " $ \(item)")
+      }
+    }
+
+    try entry.checkPaths()
+    try entry.run(parserCommand: cmd)
+  }
+
+  /// Flatten all commands and their corresponding entries into 1 list
+  func collectCommandsAndEntries() throws -> [(ParserCLI, ConfigEntry)] {
+    try commands.keys.sorted()
+      .compactMap { cmd in
```

Comment:
Why not just `map`?
If I'm not mistaken, that closure takes a string and returns a (non-optional) Parser, right?
(And if a parser can't be found it `throws`, exiting from the whole method directly not just the single iteration on that particular cmd anyway)

Reply:
Probably a left-over from getting type errors for the 2nd closure.